ee/dataflatten/flatten.go

```diff
@@ -140,8 +147,8 @@ func Flatten(data interface{}, opts ...FlattenOpts) ([]Row, error) {
 		opt(fl)
 	}
 
-	if !fl.debugLogging {
-		fl.logger = level.NewFilter(fl.logger, level.AllowInfo())
+	if fl.debugLogging {
+		fl.logLevel = slog.LevelDebug
 	}
```

Comment:
I suspect we could drop this, and simply log _everything_ at at `levelFlattenDebug` and let the caller apply the appropriate filter. But 🤷 maybe better to merge

Reply:
I liked this because it works the same way it used to 🤷‍♀️